### DISCUSSION TITLE
scikit-rf in ANSYS PyAEDT

### DISCUSSION THREAD

**Discussion by jhillairet** [scikit-rf/scikit-rf#856] [General]
Interesting news: scikit-rf is now included in the (full) requirements in ANSYS [PyAEDT](https://github.com/pyansys/pyaedt), the Python library that interacts directly with ANSYS Electronics products (like HFSS etc) and which is currently developed by ANSYS.

https://github.com/pyansys/pyaedt/pull/2556

Currently, the method [`get_touchstone_data`](https://aedt.docs.pyansys.com/release/0.6/API/_autosummary/pyaedt.hfss3dlayout.Hfss3dLayout.get_touchstone_data.html#get-touchstone-data) is returning a `Network` like object ;-)

It seems some of the customer of ANSYS have openned some issues here (and some not solved yet, like #741 ).

**Comment by cafeclimber** [comment]
I've used / am using pyaedt for my PhD research and this is _wonderful_ news! 🎉 🚀 

Time to keep pushing towards 1.0!
